module/VuFind/tests/unit-tests/src/VuFindTest/CSV/ImporterTest.php

```diff
@@ -67,6 +68,10 @@ public function setUp(): void
     {
         $this->csvFixtureDir = $this->getFixtureDir() . 'csv/';
         $this->container = new MockContainer($this);
+        $this->container->set(
+            PathResolver::class,
+            new PathResolver(APPLICATION_PATH, [])
```

Comment:
I suppose we could use the factory here for consistency with other tests, if we go down that road (though don't take all these related comments as insistence that we need to -- just want to document everything in case we decide that way).